src/index.html

```diff
@@ -1,13 +1,43 @@
 <!doctype html>
+<%
+  const mapToHtml = (filteredChunks, chunks) => filteredChunks.map(chunk => `<script type="text/javascript" src="${chunks[chunk].entry}"></script>`).join('\n    ');
+
+  const headChunks = (headChunks, chunks) => {
+    if (!headChunks) {
+      return '';
+    }
+    return mapToHtml(headChunks, chunks);
+  };
+
+  const bodyChunks = (headChunks, chunks) => {
+    let body = Object.keys(chunks);
+
+    if (headChunks) {
+      body = body.filter(chunk => !headChunks.includes(chunk));
+    }
+
+    if (!body) {
+      return 'undefined';
```

Comment:
should this be undefined or empty string like above?

Reply:
yeah, it should be consistent.